tests/test_config_parser.py

```diff
@@ -166,7 +171,7 @@ def __eq__(self, other: Mapping[str, Any]) -> bool:
                 is_eq = False
         return is_eq
 
-    def _append_list(self, ref_entry, new_args, job_name=DEFAULT_JOB_NAME, separator=","):
+    def _append_list(self, ref_entry: str, new_args: Any, job_name: str = DEFAULT_JOB_NAME, separator: str = ",") -> None:
```

Comment:
`new_args` is either a `List[str]` or `str`. Went with `Any` like the [docs said](https://mypy.readthedocs.io/en/stable/common_issues.html#redefinitions-with-incompatible-types)

Reply:
When something can be one of two types, you can use `Union[str, List[str]]` to limit the type to only those two types